src/main/java/com/jensjansson/ce/bot/BotAvatarUtil.java

```diff
@@ -35,6 +36,7 @@ private static void updateAvatars(TopicConnection topic,
             SerializableFunction<Stream<UserInfo>, Stream<UserInfo>> updater) {
         CollaborationMap map = getMap(topic);
         List<UserInfo> oldUsers = map.get(MAP_KEY, LIST_USER_INFO_TYPE_REF);
+        oldUsers = oldUsers != null ? oldUsers : Collections.emptyList();
```

Comment:
I suspect that this means that that the `oldUsers` that is passed to `map.replace` will not match what's in the map in case it was empty, which in turn means that the initial update would end up in an infinite loop?

Reply:
Fixed